### PR TITLE
Update configuration docs from json to toml

### PR DESCRIPTION
**Pull request by penguwin**
This updates the documentation about the configuration system from `json` to `toml`. This is relevant for https://github.com/knoxite/knoxite/pull/173

### CODE REVIEW

**Review by ma-hartma — APPROVED** [review]

File: content/docs/configuration-system/content.md [hunk 1]
```diff
@@ -43,10 +55,8 @@ $ knoxite config init
...
 This will leave us with an empty configuration file:
-```json
-{
-    "repositories": {}
-}
+```toml
+[repositories]
 ```
 
 By default knoxite will use the OS dependant standard paths for this file:
```
Comment (ma-hartma) [inline content/docs/configuration-system/content.md:62]: depend**e**nt dominates in American English

File: content/docs/configuration-system/content.md [hunk 2]
```diff
@@ -67,36 +77,29 @@ password part and knoxite will prompt you for it - this is also considered more
...
 Another cool thing is that knoxite will recognize encrypted configuration files
 while trying to read them out. Therefore you can reference them just like a
-normal configuration file - without the `crypto://` protocol prefix - for
-further usage.
+normal configuration file - without the `crypto://` protocol prefix - in further
+usage.
 
 ### Creating an alias
-The `alias` command creates a shorthand for a repository's configuration. In
+The `alias` command creates a shorthand for a repositories configuration. In
```
Comment (ma-hartma) [inline content/docs/configuration-system/content.md]: repository's is the singular

**Review by ma-hartma — CHANGES_REQUESTED** [review]

File: content/docs/configuration-system/content.md (hunk 1 above)
Comment (penguwin) [reply content/docs/configuration-system/content.md:62]: using the same version as in our code comment

**Review by ma-hartma — APPROVED** [review]
LGTM!

**Review by muesli — APPROVED** [review]
LGTM!

**Pull request merged**